src/main/java/org/maxgamer/quickshop/QuickShop.java

```diff
@@ -1574,6 +1574,11 @@ private void updateConfig(int selectedVersion) {
             getConfig().set("config-version", 106);
             selectedVersion = 106;
         }
+        if (selectedVersion == 106) {
+            getConfig().set("use-enchantment-for-enchanted-book", false);
```

Comment:
You should use "shop.use-enchantment-for-enchanted-book" :D

Reply:
Yep. Totally correct.

Also fixed the 'Mending I' type of issues.